### ISSUE TITLE
Hide on Mobile doesn't work in pt-BR language

### ISSUE THREAD

**Issue by agnieszkakowalska** [EDMdesigner/editor-issues#81]
Can you please fix this? Our client can't use this feature.

![obraz](https://cloud.githubusercontent.com/assets/17157918/26344105/bf9e0c10-3f9e-11e7-8318-f2d61755421e.png)

**Comment by smiska** [comment]
Hi Agnieszka,

The buttons are not working because the current labels are too long. These labels are "Ligado" and "Desligado". Does your client have a desired translation? We can provide a quicker fix if you could provide the labels. 
If I'm not mistaken, this is "pt" localization not "pt-Br". For pt-Br the labels are working and they are "Sim" and "Não". Would it be a proper translation here as well?

**Comment by agnieszkakowalska** [comment]
You're right, I was referring to pt localization. I've just checked and I can't see any pt-Br language in the pack. After listing editor.availableLanguages I get: `["de", "en", "hu", "it", "nl", "ro", "ru", "fr", "pl", "ru", "zh-HANS", "pt", "es", "cs"]` Can you tell me how to get pt-br localization?

**Comment by agnieszkakowalska** [comment]
Thanks, let me consult it with a client.

**Comment by agnieszkakowalska** [comment]
Thank you, we'll just change from pt to pt-br on our side.

**Comment by smiska** [comment]
So, you found the pt-br option? The language update won't be necessary then? Please tell me if you need assistance of any kind.

**Comment by agnieszkakowalska** [comment]
Yes, I have it. Anyway language update would be nice because pt version doesn't work correctly right now, however you don't need to rush. Thank you for your help.

**Comment by smiska** [comment]
Just to be clear, by the fixing the 'pt' version you mean to fix the mobile settings toolbox as seen in the screenshot you provided. Or are there other updates necessary? I'm not sure if you confirmed, are the "Sim" and "Não" translations correct for the update? Thanks for the help.

**Comment by agnieszkakowalska** [comment]
We switched to pt-br instead of pt so everything is fine on our side. You can shorten the pt translation so they are displayed correctly but I am not sure what will be the best translation here. Thanks.

**Comment by smiska** [comment]
Hi, I can only continue with the update of the localization labels, if you provide me translations that are conventional and widely used among native users and approved as a correct translation. Please if it isn't much trouble ask your users what labels should I implement. Thanks.
